src/vivarium/framework/values.py

```diff
@@ -365,14 +367,14 @@ def register_value_modifier(self,
 
         """
         modifier_name = self._get_modifier_name(modifier)
-        logger.debug(f"Registering {modifier_name} as modifier to {value_name}")
 
         pipeline = self._pipelines[value_name]  # May create a pipeline
         pipeline.mutators.append(modifier)
 
         name = f'{value_name}.{len(pipeline.mutators)}.{modifier_name}'
+        logger.debug(f"Registering {name} as modifier to {value_name}")
```

Comment:
Log the resource name rather than the modifier name.  Resouce name has info about what pipeline it modifies and what order it gets called in.